tests/gis_tests/test_geoip2.py

```diff
@@ -22,8 +22,7 @@ class GeoIPTest(SimpleTestCase):
     addr = '128.249.1.1'
     fqdn = 'tmc.edu'
 
-    def test01_init(self):
-        "GeoIP initialization."
+    def test_initialization(self):
```

Comment:
Number prefix could be done separately as other files are affected. (And I think `init` (method name) is fine rather than `initialization`.)